### ISSUE TITLE
Missing file

### ISSUE THREAD

**Issue by ghost** [freetnt5852/LadyBug-Bot#54]
You have to add the .env file by yourself. 

Required:

TOKEN_DEV= 
TOKEN =
DEV=
CLIENT_ID=

**Comment by ghost** [comment]
I don't know why this file isn't in the category. But this guy needs to update the files fr.

**Comment by ravener** [comment]
Yeah I forgot to add this at the start, but then it wasn't my intention to allow others to self-host this so I just never bothered. I guess I'll add it soon anyway

**Comment by itsMathew** [comment]
For this you should typically create a .env.example file that users can turn into a .env file.. lmk if I can create a PR for it.. pretty small thing tbh

**Comment by ravener** [comment]
> For this you should typically create a .env.example file that users can turn into a .env file.. lmk if I can create a PR for it.. pretty small thing tbh

yup i usually do put a .env.example in my projects, just not this time i guess, but i'm currently unsatisfied with the state of the bot as it is and there's a typescript rewrite in progress that i've been procrastinating and that'll solve a lot of problems
